src/applications/disability-benefits/view-payments/components/view-payments-lists/payments/Payments.jsx

```diff
@@ -1,12 +1,12 @@
-import React, { useEffect, useState, useRef } from 'react';
-import { chunk } from 'lodash';
 import PropTypes from 'prop-types';
+import React, { useCallback, useEffect, useState, useRef } from 'react';
 import { VaPagination } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import { chunk } from 'lodash';
 import { useLocation, useNavigate } from 'react-router-dom-v5-compat';
 
-import { clientServerErrorContent } from '../helpers';
+import { clientServerErrorContent, useResizeObserver } from '../helpers';
 
-const MAX_PAGE_LIST_LENGTH = 10;
+const MAX_PAGES_CONTAINER_WIDTH = 640; // USWDS width-tablet setting
```

Comment:
In the original issue, we started seeing issues when the container width fell below ~500 pixels with 10 pagination links. I didn't want to have a random magic number in the code, so I fell back on the USWDS breakpoint that was closest to this number: https://designsystem.digital.gov/utilities/height-and-width/

Reply:
Could we pull the small/large screen page length numbers up here too?